### ISSUE TITLE
More Docs

### ISSUE THREAD

**Issue by ColeSquared** [Sceleratis/Adonis#10]
When creating a plugin for matchmaking band in my game, I had to dig through the `MainModule` just to find the Trello parts, then find the Trello module (you should make that open source too so we don’t have to dig through Adonis for it), and finally finding how to detect player names (which I’m probably not doing right), I finally was able to put together a small plugin with a matchmaking ban command. 

Tl;dr Adonis should be better documented for plugins and the API.

**Comment by Dog2puppy** [comment]
Docs could be done through either GitBook (yah) or GitHub wikis (nah).

**Comment by Sceleratis** [comment]
Yeah I know it needs better documentation for a lot of stuff, thought going back through and doing all that might prove a little difficult considering I lack time currently for most things. I started to somewhat document functions in the API module included with the Adonis loader but the documentation in that should be massively outdated now..

**Comment by Dog2puppy** [comment]
I could document a couple of the simpler parts that I understand in Adonis.

**Comment by Dog2puppy** [comment]
GitHub would work.

**Comment by DavidBaszuckiA** [comment]
good news, this is  becoming an Premium Adonis feature, 34.99$ a month, but because your the lucky suggester, we'll get you a free 4 day free trial!

**Comment by ColeSquared** [comment]
Cool joke

**Comment by happyman090** [comment]
> good news, this is becoming an Premium Adonis feature, 34.99$ a month, but because your the lucky suggester, we'll get you a free 4 day free trial!

When you can't even place the dollar sign in the correct position 🤡
Also, "an" is only used when the next word begins with a vowel (or is pronounced so that it sounds like one).
And it's "you're" in this case, not "your" 😐. Now, quit trying to impersonate people who don't exist and start getting a life, ya poor plebe 😮‍💨


in regards to the actual issue at hand, i _might_ make docs for some parts of the api when I have time

**Comment by GalacticInspired** [comment]
I am interested for the 4 day trial.

**Comment by DaEnder** [comment]
> good news, this is becoming an Premium Adonis feature, 34.99$ a month, but because your the lucky suggester, we'll get you a free 4 day free trial!

do the mouth shut

**Comment by Sceleratis** [comment]
I would mention I am working on this (as of today)
